hardening-check.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: hardening-check
+  version: 2.23.7
+  epoch: 0
+  description: "Debian devscripts hardening-check"
+  copyright:
+    - license: GPL-2.0-or-later
+  dependencies:
+    runtime:
+      - binutils
+
+environment:
+  contents:
+    packages:
+      - busybox
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://salsa.debian.org/debian/devscripts.git
+      tag: v${{package.version}}
+      expected-commit: eb7dc80e72486f7955ec6793a8067fe69fda7883
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/usr/bin
+      mv scripts/hardening-check.pl "${{targets.destdir}}"/usr/bin/hardening-check
+
+update:
+  # Last change 2 years ago. It only makes sense to update when there
+  # is a manjor toolchain evolution.
+  enabled: false
```

Comment:
even if it's not updated upstream very often it's still probably worth tracking the version. It's on release monitoring https://release-monitoring.org/project/5419/